### PR TITLE
document changes in bind & replacements

### PR DESCRIPTION
**Pull request by ephys**
This PR documents https://github.com/sequelize/sequelize/pull/14447

It also clarifies that `bind` & `replacements` can be used in `literal()`

It also changes a few uses of `sequelize.fn` and the like with `fn`. Doing that removes a lot of noise from queries, and removes the assumption that users need to pass their `sequelize` instance around to use these functions.

### CODE REVIEW

File: docs/other-topics/upgrade.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ Head to our [Versioning Policy page](/releases) to see exactly which databases a
 
 ### Blocking access to `/lib`
 
+*Pull Request [#14352]*
```
Comment (WikiRik) [inline docs/other-topics/upgrade.md:43]: Not so sure if we need this, but it can't hurt to add

File: docs/core-concepts/raw-queries.md [hunk 2]
```diff
@@ -115,75 +124,125 @@ await sequelize.query(
...
+
+await sequelize.query(
+  'SELECT * FROM projects WHERE id = CAST($1 AS int)',
+  {
+    bind: [5],
+    type: QueryTypes.SELECT,
+  },
+);
+
+// some dialects (such as postgres) support a terser cast syntax you can use if you prefer:
```
Comment (WikiRik) [inline docs/core-concepts/raw-queries.md:246]: Should we make this a regular text with SQL example?

**Review by WikiRik — APPROVED** [review]
Looks good! Comment is non-blocking

File: docs/core-concepts/raw-queries.md (hunk 2 above)
Comment (ephys) [reply docs/core-concepts/raw-queries.md]: Done

File: docs/other-topics/upgrade.md (hunk 1 above)
Comment (ephys) [reply docs/other-topics/upgrade.md:43]: For large or controversial changes, it can give more information explaining why the change was done

**Review by WikiRik — APPROVED** [review]

**Pull request merged**